### PR TITLE
Fix and improve README

### PR DESCRIPTION
**Pull request by ctjhoa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,61 @@
...
+src/main.rs:9         Some(y) => println!("{:?}", y),
+src/main.rs:10         _ => ()
+src/main.rs:11     }
+```
+
+You can add `OPTIONS` to `allow`/`warn`/`deny`:
+- the whole set using the `clippy` lint group (`#[deny(clippy)]`, etc)
+- only some lints (`#[deny(single_match, box_vec)]`, etc)
+
+*`deny` produce error instead of warnings*
```
Comment (Manishearth) [inline README.md]: nit: *produces

**Comment by Manishearth** [comment]
Thanks! Minor nit, and it should be ready to merge!

**Comment by ctjhoa** [comment]
@Manishearth @llogiq Any chance to merge ? :'(

File: README.md [hunk 2]
```diff
@@ -31,10 +31,50 @@ To use, add the following lines to your Cargo.toml:
...
+git = "https://github.com/Manishearth/rust-clippy"
+```
+
+Sample `main.rs`:
+```rust
+#![feature(plugin)]
 
-You can allow/warn/deny the whole set using the `clippy` lint group (`#[allow(clippy)]`, etc)
+#![plugin(clippy)]
+// OPTIONS GO HERE
```
Comment (Manishearth) [inline README.md]: nit: either remove this line or explain how the options work above

**Comment by Manishearth** [comment]
I'll fix that nit directly.
Thanks!

**Pull request merged**